ABloom/ABloom/Presentation/Main/QnAList/QnAListView.swift

```diff
@@ -153,9 +153,17 @@ extension QnAListView {
     SignInView(activeSheet: activeSheet)
       .presentationDetents([.height(302)])
       .onDisappear {
-        Task { await qnaListVM.fetchDataAfterSignIn() }
+        Task { await qnaListVM.fetchDataAfterSignIn()
+          await qnaListVM.checkNeedSignUp() }
```

Comment:
```suggestion
          await qnaListVM.checkNeedSignUp()
          if qnaListVM.currentUser  == nil { // 시트 띄우기}
          else { // 넘어가기 }
          }
```
이런식으로 줄일 수 있지 않을까요?
checkNeedSignUp() 함수에서 또 불필요한 호출과 연산이 생긴 듯 합니다.
이미 fetchDataAfterSignIn에서 데이터를 불러왔기 때문에 이를 활용하는 게 좋아보여요 😀

Reply:
홈뷰에서는 작동하는데, 다른 뷰에서 로그인하러 홈에 들어오면 작동하지 않더라구요 ㅠㅠ
vm에 있는 변수를 조절하면 다른 함수들 호출은 불필요해질것 같습니다! 다시 하고 올릴게요!!